consensus/src/chained_bft/block_storage/block_store.rs

```diff
@@ -254,24 +254,18 @@ impl<T: Payload> BlockStore<T> {
     }
 
     async fn execute_block(&self, block: Block<T>) -> anyhow::Result<ExecutedBlock<T>> {
-        let parent_block = match self.verify_and_get_parent(&block) {
-            Ok(t) => t,
-            Err(e) => {
-                security_log(SecurityEvent::InvalidBlock)
```

Comment:
I don't think we should log at the block store level, and instead log invalid blocks in:

* process_proposal (which we do)
* network.request_block() (which we don't, but we can fix this)